comps/dataprep/minio/milvus/langchain/prepare_doc_milvus.py

```diff
@@ -0,0 +1,514 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import io
+import json
+import os
+import tempfile
+from pathlib import Path
+from typing import List, Optional, Union
+
+import msgpack
+from config import (
+    COLLECTION_NAME,
+    LOCAL_EMBEDDING_MODEL,
+    MILVUS_HOST,
+    MILVUS_PORT,
+    MINIO_ACCESS_KEY,
+    MINIO_DOCUMENT_BUCKET,
+    MINIO_ENDPOINT,
+    MINIO_SECRET_KEY,
+    MINIO_SECURE,
+    MOSEC_EMBEDDING_ENDPOINT,
+    MOSEC_EMBEDDING_MODEL,
+    TEI_EMBEDDING_ENDPOINT,
+)
+from fastapi import Body, File, Form, HTTPException, Request, UploadFile
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain_community.embeddings import HuggingFaceBgeEmbeddings, HuggingFaceHubEmbeddings, OpenAIEmbeddings
+from langchain_core.documents import Document
+from langchain_milvus.vectorstores import Milvus
+from langchain_text_splitters import HTMLHeaderTextSplitter
+from minio import Minio, S3Error
+
+from comps import CustomLogger, DocPath, opea_microservices, register_microservice
+from comps.dataprep.minio.milvus.langchain.config import MINIO_WAREHOUSE_BUCKET
+from comps.dataprep.minio.minio_schema import MinioEventNotification
+from comps.dataprep.utils import (
+    create_upload_folder,
+    decode_filename,
+    document_loader,
+    encode_filename,
+    get_separators,
+    get_tables_result,
+    parse_html,
+    remove_folder_with_ignore,
+)
+
+logger = CustomLogger("prepare_doc_minio_milvus")
+logflag = os.getenv("LOGFLAG", True)
+
+# workaround notes: cp comps/dataprep/utils.py ./milvus/utils.py
+# from utils import document_loader, get_tables_result, parse_html
+index_params = {"index_type": "FLAT", "metric_type": "IP", "params": {}}
+partition_field_name = "filename"
+upload_folder = "./uploaded_files/"
```

Comment:
FYI: Python naming convention is to use uppercase for (global) constants:
https://peps.python.org/pep-0008/#naming-conventions

Reply:
I do not have an option to resolve this fixed item.  Could somebody else resolve this?